README.md

```diff
@@ -8,6 +8,8 @@ This repository hosts two different projects:
 * [Broker microservice](./packages/openactive-broker-microservice/): this sits in between the test suite and the target Open Booking API implementation. This allows the test suite to watch for changes, and throws them back to it.
 * [Integration tests](./packages/openactive-integration-tests): this performs automated tests against the API
 
+[The Roadmap](./ROADMAP.md) provides an overview of upcoming milestones and when they're expected to be completed.
```

Comment:
Perhaps:

"provides an overview of upcoming development milestones, and contributions are welcome"

To put the focus on community involvement?

Reply:
Done